pkg/test/test_data/crd-in-step/00-assert.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: mycrds.mycrd.k8s.io
+  name: mycrds.mycrd.example.com
```

Comment:
Just curious why did these need to change?
Can you put a comment into PR description?